jwst/datamodels/schemas/fgs.schema.yaml

```diff
@@ -0,0 +1,132 @@
+1allOf:
+- $ref: core.schema.yaml
+- $ref: bunit.schema.yaml
+- type: object
+  properties:
+    data:
+      title: The output science data
+      fits_hdu: SCI
+      default: 0.0
+      ndim: 4
+      datatype: float32
+    dq:
+      title: Data quality array
+      fits_hdu: DQ
+      default: 0
+      ndim: 2
+      datatype: uint32
+    planned_star_table: 
+      title: Planned reference star table
+      fits_hdu: PLANNED_STARS
+      datatype:
+      - name: guide_star_order
+        datatype: int32
+      - name: reference_star_id
+        datatype: string
+      - name: ra
+        datatype: float64
+      - name: dec
+        datatype: float64
+      - name: id_x
+        datatype: float64
+      - name: id_y
+        datatype: float64
+      - name: fgs_mag
+        datatype: float64
+      - name: fgs_mag_uncert
+        datatype: float64
+      - name: count_rate
+        datatype: float64
+      - name: count_rate_uncert
+        datatype: float64
+    flight_star_table: 
+      title: Planned reference star table
```

Comment:
Looks like the title of this one should not be "Planned", same as the one above, but should be something like "Flight".